scripts/presubmit-tests.sh

```diff
@@ -20,111 +20,206 @@
 source $(dirname ${BASH_SOURCE})/library.sh
 
 # Extensions or file patterns that don't require presubmit tests.
-readonly NO_PRESUBMIT_FILES=(\.md \.png ^OWNERS ^OWNERS_ALIASES)
+readonly NO_PRESUBMIT_FILES=(\.md \.png \.gitignore \.gitattributes ^OWNERS ^OWNERS_ALIASES ^AUTHORS)
```

Comment:
should `\.md` be removed from this list?

Reply:
Argh, rebasing copy-pasta. Fixed, thanks.